### DISCUSSION TITLE
Odd dailystats values

### DISCUSSION THREAD

**Discussion by emale999** [ZuinigeRijder/hyundai_kia_connect_monitor#45] [Q&A]
Hi there,

I get very odd dailystat values. Look here at the csv - I added a kWh/100km column at the end:
![Bildschirmfoto 2023-02-15 um 08 37 33](https://user-images.githubusercontent.com/12543734/218962825-666700bb-67d6-4a21-911a-97f1cc6dc811.png)

This isn't a problem with your scripts - I also get power consumption values like 70000Wh for a trip of 29km (that's more or less the whole battery content!) with other scripts that connect to bluelink.
And I defenetively didn't have to charge from 0 to 100% after that 29km trip on 16.1.2023. On board system displays a correct average consumption of 18-19 kWh/100km. Here are the statistics from the onboard system - telling something totally different as the bluelink data (the displayed 10.6 and 2.2 kWh/100km seem oddly but are correct because they were completely down a montain) 11.2. corresponds to bluelink API, but 12.+13.2. do not.
![Bildschirmfoto 2023-02-15 um 08 54 48](https://user-images.githubusercontent.com/12543734/218985766-a9d2a696-e6f9-47c2-a2c7-6fee65b474d8.png)

Do you have any experience with such odd values? Values like "total_consumed" defenetively are in Wh, isn't it?

Best regards
emale999

**Comment by ZuinigeRijder** [comment]
These are surely strange values. Maybe some questions first:
- which car do you drive?
- which infotainment/software version is installed in your car? 
- I assume you live in Europe (I think Germany looking at the screenshot ;-)

I remember that earlier infotainment versions in the car gave wrong consumption figures. Do you still have an old infotainment version in the car?

I looked up my values and they are as expected. They go back till December 2022. And yes, these values like total_consumed are in Wh.

**Comment by emale999** [comment]
It's an IONIQ5 in the EU. 

I will check the infotainement software version next time I get to the car, but that's a good lead that you remember that earlier infotainment versions in the car gave wrong consumption figures. Maybe that's the problem here. The IONIQ5 now is one year old. Only got a software update last summer because of a recall campaign concerning a handbrake issue. But that update maybe didn't apply to the infotainement system.
Maybe it's really time to make an appointment at the garage - HeadUnit also needs an update because battery preheating was added some weeks ago.

However it's strange that the onboard system shows other consumption values than it submits to the bluelink server.

**Comment by ZuinigeRijder** [comment]
You can install the newest 221223 version yourself with an USB stick or SSD of 64 GB or bigger.
See [here for the update website](https://update.hyundai.com/EU/DE/Main).

See [this forum about German discussion about this version](https://www.goingelectric.de/forum/viewtopic.php?f=535&t=84387).

**Comment by emale999** [comment]
I know. But I also need the HeadUnit update only feasible by the garage as far as I know.  ;-)

**Comment by ZuinigeRijder** [comment]
The update site will update the Maps AND the infotainment version. The latter is the HeadUnit you refer to, I think. Only for firmware updates of sensors you need to go to the garage,
